libraries/scopes/src/simplify.js

```diff
@@ -0,0 +1,58 @@
+const { normalizeScopeSet, scopeCompare } = require("./normalize");
+
+exports.simplifyScopeExpression = scopeExpression => {
+  let key = Object.keys(scopeExpression);
```

Comment:
What if `scopeExpression` is just a scope, like `"scope1"`?

Reply:
Yes, I think my approach here is really flawed. It's better for me to rethink the whole function by taking in the points you said.

Reply:
Yep, "refactoring" is an important part of the TDD process :)

Reply:
"Flawed" sounds mean -- it passes the tests!  It's just a step along the way, part of the process :)

Reply:
Oh, :). But I hope someday I will be able to write a good solution to this problem. I'll keep on working.